core/src/test/java/org/apache/iceberg/rest/Route.java

```diff
@@ -131,7 +131,12 @@ enum Route {
       FetchScanTasksRequest.class,
       FetchScanTasksResponse.class),
   CANCEL_PLAN_TABLE_SCAN(
-      HTTPRequest.HTTPMethod.DELETE, ResourcePaths.V1_TABLE_SCAN_PLAN, null, null);
+      HTTPRequest.HTTPMethod.DELETE, ResourcePaths.V1_TABLE_SCAN_PLAN, null, null),
+  LOAD_TABLE_CREDENTIALS(
```

Comment:
now the loadCreds supports adding `?=plan-id` in query param can we include that ?

Reply:
Yep, I think I've added that properly. Great comment, thanks!